tests/test_knowledge/test_clause.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+
+from doggos.knowledge import LinguisticVariable, Domain
+from doggos.knowledge import Clause
+from doggos.fuzzy_sets import Type1FuzzySet
+from doggos.knowledge.linguistic_variable import LinguisticVariable, Domain
+
+import numpy as np
+
+
+class TestClause:
+
+    def test_exception_typeerror_lingvar_init(self):
+        fuzzy_set = Type1FuzzySet(lambda x: 0 if x < 0 else 1)
```

Comment:
Change this lambda to something else, cause it's error prone and we don't want users to make such functions.